.junie/guidelines.md

```
@@ -0,0 +1,87 @@
+| ライブラリ                                                          | 用途                     |
+|----------------------------------------------------------------|------------------------|
+| [jwt-go](https://github.com/golang-jwt/jwt)                    | JSON Web Token         |
+| [uuid](https://github.com/google/uuid)                         | UUID                   |
+| [grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway) | gPRCからJSONへのプロキシジェネレータ |
+| [pq](https://github.com/lib/pq)                                | postgresドライバー          |
+| [paseto](https://github.com/o1egl/paseto)                      | PASETO Token           |
+| [viper](https://github.com/spf13/viper)                        | 設定ファイル読み込みなど           |
+| [crypt](https://github.com/golang/crypto)                      | 暗号化                    |
+| [go-genproto](https://github.com/googleapis/go-genproto)       |                        |
+| [grpc-go](https://github.com/grpc/grpc-go)                     | gRPC                   |
+| [protobuf]()                                                   | Protocol Buffers       |
```

markdownlint-cli2 (0.17.2)

No empty links
null

(MD042, no-empty-links)